.github/workflows/wheezy_tests.yml

```diff
@@ -0,0 +1,51 @@
+name: Wheezy CI
+on:
+  pull_request:
+    paths:
+      - "Formula/patchelf.rb"
+      - "Formula/binutils.rb"
+jobs:
+  wheezy_tests:
+    if: github.repository == 'Homebrew/homebrew-core' && github.event_name == 'pull_request' && ! contains(github.event.pull_request.labels.*.name, 'CI-syntax-only')
+    runs-on: ubuntu-latest
+    env:
+      HOMEBREW_CORE_GIT_REMOTE: ${{github.event.repository.html_url}}
+      HOMEBREW_FORCE_HOMEBREW_ON_LINUX: 1
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+      - name: Hack curl version
+        run: |
+          sed -i 's/HOMEBREW_MINIMUM_CURL_VERSION="7.41.0"/HOMEBREW_MINIMUM_CURL_VERSION="7.25.0"/g' /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/brew.sh
```

Comment:
Can we make this an undocumented environment variable instead in Homebrew/brew (that also requires e.g. `HOMEBREW_FORCE_HOMEBREW_ON_LINUX` and `HOMEBREW_DEVELOPER` to be set)?

Reply:
I can do that. I'll open a pull request later today.